src/org/usfirst/frc/team467/robot/Grabber.java

```diff
@@ -91,23 +105,25 @@ public void periodic() {
 
 		if (!RobotMap.useSimulator) {
 			left.set(speed);
-			right.set(-speed);
+			right.set(-1 * speed);
```

Comment:
This is a regression. Just keep the minus sign in front, don't multiply by negative one.